File: pingyougou_parent/pinyougou_sellergoods_interface/src/main/java/com/pinyougou/sellergoods/service/BrandService.java
package com.pinyougou.sellergoods.service;

import com.pinyougou.pojo.TbBrand;
import entity.PageResult;

import java.util.List;
import java.util.Map;

public interface BrandService {

    public List<TbBrand> findAll();

    PageResult findPage(int pageNum, int pageSize);

    void add(TbBrand tbBrand);

    TbBrand findOne(Long id);

    void update(TbBrand tbBrand);

    void dele(Long[] ids);

    PageResult search(int pageNum, int pageSize, TbBrand tbBrand);

    List<Map> findBrandList();

}

File: pingyougou_parent/pinyougou_manager_web/src/main/webapp/js/service/brandService.js
app.service("brandService", function ($http) {
    this.findBrandList=function () {
        return $http.get("../brand/findBrandList");
    }
    //查询所有
    this.findAll = function () {
        return $http.get("../brand/findAll");
    };
    //根据条件查询数据
    this.search = function (pageNum, pageSize, searchEntity) {
        return $http.post("../brand/search/" + pageNum + "/" + pageSize, searchEntity);
    };
    //添加数据
    this.add = function (entity) {
        return $http.post("../brand/add",entity);
    };
    this.update = function (entity) {
        return $http.post("../brand/update",entity);
    };
    //    修改参数
    this.findOne = function (id) {
        return $http.get("../brand/findOne/" + id);
    };
    //    删除数据
    this.dele = function (selectIds) {
        //判断selectIDs数组中的数据是否为空
        return $http.get("../brand/dele/"+selectIds);
    }
})